core-bundle/src/Resources/config/voters.yml

```
@@ -0,0 +1,14 @@
+services:
+    _defaults:
+        autoconfigure: true
+
+    # Backwards Compatibility
+    contao.security.backend_access_voter: '@Contao\CoreBundle\Security\Voter\BackendAccessVoter'
+
+    Contao\CoreBundle\Security\Voter\ContentModelAccessVoter:
```

```suggestion
    Contao\CoreBundle\Security\Voter\ContentModelAccessVoter: ~
```

I prefer to be explicit, but in the end it is just a matter of personal preference.

---

yeah I used that too, until I found it it's not necessary. Honestly, I'd love to autowire these, there's no DI so 🤷‍♂ 😂

---

Go for it. 👍

---

Technically you don't need to enable autowiring in order to use autoregistration.